README.md

```diff
@@ -722,6 +725,15 @@ Please note that the default for the [attribute](#attributes) is: `false`.
 
 You can change your default setting in your [configuration](#configuration) with the key `defaults.remove_image`.
 
+### Excluded heading
+
+You can exclude certain headings in the TOC by adding a special comment to the line before the heading, as shown below.
```

Comment:
Suggestion: "You can exclude certain headings in the TOC by adding a special comment to the line above the line with the heading, as shown below."